### PR TITLE
⭐️ improve MQL resource docs description

### PR DESCRIPTION
**Pull request by chris-rock**
fixes https://github.com/mondoohq/cnquery/issues/1049

### CODE REVIEW

**Comment by tas50** [comment]
Docs look to be generating well now

**Review by misterpantz — APPROVED** [review]
Thank you!

File: Makefile [hunk 1]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
 .PHONY: lr/docs/markdown
 lr/docs/markdown: lr/build
 	./lr markdown resources/packs/aws/aws.lr \
-        --pack-name "Amazon Web Services (AWS)" \
+    --pack-name "Amazon Web Services (AWS)" \
+		--description "The Amazon Web Services (AWS) resource pack provides resources that let you query and assess the security of your Amazon Web Services (AWS) cloud services." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Amazon Web Services (AWS) resource pack provides resources that let you query and assess the security of your AWS cloud services." \
```

File: Makefile [hunk 2]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/aws/aws.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/aws-pack
 	./lr markdown resources/packs/azure/azure.lr \
 		--pack-name "Azure" \
+		--description "The Azure resource pack provides resources that let you query and assess the security of your Azure cloud services." \
 		--docs-file resources/packs/azure/azure.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/azure-pack
 	./lr markdown resources/packs/core/core.lr \
 		--pack-name "Core" \
+		--description "The Core pack provides basic resources that let you query and assess the security." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Core resource pack provides basic resources that let you query and assess the security of your infrastructure." \
```

File: Makefile [hunk 3]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 lr/docs/markdown: lr/build
 	./lr markdown resources/packs/aws/aws.lr \
-        --pack-name "Amazon Web Services (AWS)" \
+    --pack-name "Amazon Web Services (AWS)" \
+		--description "The Amazon Web Services (AWS) resource pack provides resources that let you query and assess the security of your Amazon Web Services (AWS) cloud services." \
 		--docs-file resources/packs/aws/aws.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/aws-pack
 	./lr markdown resources/packs/azure/azure.lr \
 		--pack-name "Azure" \
+		--description "The Azure resource pack provides resources that let you query and assess the security of your Azure cloud services." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Azure resource pack provides resources that let you query and assess the security of your Azure cloud services." \
```

File: Makefile [hunk 4]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/azure/azure.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/azure-pack
 	./lr markdown resources/packs/core/core.lr \
 		--pack-name "Core" \
+		--description "The Core pack provides basic resources that let you query and assess the security." \
 		--docs-file resources/packs/core/core.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/core-pack
 	./lr markdown resources/packs/gcp/gcp.lr \
 		--pack-name "Google Cloud Platform (GCP)" \
+		--description "The Google Cloud Platform (GCP) resource pack provides resources that let you query and assess the security of your Google Cloud Platform (GCP) cloud services." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Google Cloud Platform (GCP) resource pack provides resources that let you query and assess the security of your GCP cloud services." \
```

File: Makefile [hunk 5]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/gcp/gcp.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/gcp-pack
 	./lr markdown resources/packs/github/github.lr \
 		--pack-name "GitHub" \
+		--description "The GitHub resource pack provides resources that let you query and assess the security of your GitHub organization and repositories." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo GitHub resource pack provides resources that let you query and assess the security of your GitHub organization and repositories." \
```

File: Makefile [hunk 6]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/github/github.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/github-pack
 	./lr markdown resources/packs/gitlab/gitlab.lr \
 		--pack-name "GitLab" \
+		--description "The GitLab resource pack provides resources that let you query and assess the security of your GitLab organization and repositories." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo GitLab resource pack provides resources that let you query and assess the security of your GitLab organization and repositories." \
```

File: Makefile [hunk 7]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/gitlab/gitlab.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/gitlab-pack
 	./lr markdown resources/packs/k8s/k8s.lr \
 		--pack-name "Kubernetes (K8s)" \
+		--description "The Kubernetes resource pack provides resources that let you query and assess the security of your Kubernetes workloads." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Kubernetes resource pack provides resources that let you query and assess the security of your Kubernetes workloads." \
```

File: Makefile [hunk 8]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/k8s/k8s.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/k8s-pack
 	./lr markdown resources/packs/ms365/ms365.lr \
 		--pack-name "Microsoft 365 (MS365)" \
+		--description "The Microsoft 365 (MS365) resource pack provides resources that let you query and assess the security of your Microsoft 365 (MS365) identities and configuration." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Microsoft 365 (MS365) resource pack provides resources that let you query and assess the security of your MS365 identities and configuration." \
```

File: Makefile [hunk 9]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/ms365/ms365.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/ms365-pack
 	./lr markdown resources/packs/os/os.lr \
 		--pack-name "Operating Systems (OS)" \
+		--description "The Operating Systems (OS) resource pack provides resources that let you query and assess the security of your operating system packages and configuration." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Operating Systems (OS) resource pack provides resources that let you query and assess the security of your OS packages and configuration." \
```

File: Makefile [hunk 10]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/os/os.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/os-pack
 	./lr markdown resources/packs/terraform/terraform.lr \
 		--pack-name "Terraform IaC" \
+		--description "The Terraform IaC resource pack provides resources that let you query and assess the security of your Terraform HCL, plan and state resources." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Terraform IaC resource pack provides resources that let you query and assess the security of your Terraform HCL, plan, and state resources." \
```

File: Makefile [hunk 11]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/terraform/terraform.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/terraform-pack
 	./lr markdown resources/packs/vsphere/vsphere.lr \
 		--pack-name "VMware vSphere" \
+		--description "The VMware vSphere resource pack provides resources that let you query and assess the security of your VMware vSphere hosts and services." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo VMware vSphere resource pack provides resources that let you query and assess the security of your VMware vSphere hosts and services." \
```

File: Makefile [hunk 12]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/vsphere/vsphere.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/vsphere-pack
 	./lr markdown resources/packs/okta/okta.lr \
 		--pack-name "Okta" \
+		--description "The Okta resource pack provides resources that let you query and assess the security of your Okta identities and configuration." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Okta resource pack provides resources that let you query and assess the security of your Okta identities and configuration." \
```

File: Makefile [hunk 13]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/okta/okta.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/okta-pack
 	./lr markdown resources/packs/googleworkspace/googleworkspace.lr \
 		--pack-name "Google Workspace" \
+		--description "The Google Workspace resource pack provides resources that let you query and assess the security of your Google Workspace identities and configuration." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Google Workspace resource pack provides resources that let you query and assess the security of your Google Workspace identities and configuration." \
```

File: Makefile [hunk 14]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/googleworkspace/googleworkspace.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/googleworkspace-pack
 	./lr markdown resources/packs/slack/slack.lr \
 		--pack-name "Slack" \
+		--description "The Slack resource pack provides resources that let you query and assess the security of your Slack identities and configuration." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Slack resource pack provides resources that let you query and assess the security of your Slack identities and configuration." \
```

File: Makefile [hunk 15]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/slack/slack.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/slack-pack
 	./lr markdown resources/packs/vcd/vcd.lr \
 		--pack-name "VMware Cloud Director" \
+		--description "The VMware Cloud Director resource pack provides resources that let you query and assess the security of your VMware Cloud Director configuration." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo VMware Cloud Director resource pack provides resources that let you query and assess the security of your VMware Cloud Director configuration." \
```

File: Makefile [hunk 16]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/vcd/vcd.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/vcd-pack
 	./lr markdown resources/packs/arista/arista.lr \
 		--pack-name "Arista EOS" \
+		--description "The Arista EOS resource pack provides resources that let you query and assess the security of your Arista EOS network devices." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo Arista EOS resource pack provides resources that let you query and assess the security of your Arista EOS network devices." \
```

File: Makefile [hunk 17]
```diff
@@ -147,71 +147,88 @@ lr/docs/serve:
...
 		--docs-file resources/packs/arista/arista.lr.manifest.yaml \
 		--output ../docs/docs/mql/resources/arista-pack
 	./lr markdown resources/packs/ipmi/ipmi.lr \
 		--pack-name "IPMI" \
+		--description "The IPMI resource pack provides resources that let you query and assess the security of your IPMI devices." \
```
Comment (misterpantz) [inline Makefile]:
```suggestion
		--description "The Mondoo IPMI resource pack provides resources that let you query and assess the security of your IPMI devices." \
```

**Review by misterpantz — APPROVED** [review]
Thanks so much for making this improvement.

File: Makefile (hunk 1 above)
Comment (chris-rock) [reply Makefile]: Wouldn't it be better to call it the `The MQL Amazon Web Services (AWS) resource pack`?

Comment (misterpantz) [reply Makefile]: That's a good question. I wanted to get "Mondoo" in there for SEO. Maybe we want both "Mondoo" and "MQL" in the description?

Comment (chris-rock) [reply Makefile]: That would work too. I just think mondoo is too generic

Comment (misterpantz) [reply Makefile]: So how about "The Mondoo Amazon Web Services (AWS) resource pack lets you query and assess the security of your AWS cloud services using MQL."
?

Comment (chris-rock) [reply Makefile]: This text is confusing to me. The resource pack defines MQL and is not using MQL

Comment (misterpantz) [reply Makefile]: "using MQL" is a modifier for how "you query and assess" not for "resource pack."

Comment (misterpantz) [reply Makefile]: But I think we're into the zone of over-writing and over-analyzing.  I'll take a fresh look tomorrow.

Comment (misterpantz) [reply Makefile]: Okay it's a fresh day and I still am happy with my most recent suggestion. But if it confused you then I want to make it better!  Does this help?

"The Mondoo Amazon Web Services (AWS) resource pack lets you use MQL to query and assess the security of your AWS cloud services."

**Pull request merged**